### ISSUE TITLE
Question about Airflow version (1.10.12)

### ISSUE THREAD

**Issue by sakaia** [NetAppDocs/netapp-solutions#27]
From seeing airflow issues, it seems 1.10.12 not appropriate for airflow.

- [KubernetesExecutor single task run error: Only works with the Celery or Kubernetes executors, sorry #12341](https://github.com/apache/airflow/issues/12341)

It relates to following file.

https://github.com/NetAppDocs/netapp-solutions/blob/master/ai/aicp_hardware_and_software_requirements.adoc

**Comment by mboglesby** [comment]
@sakaia - The documentation mentions version 1.10.12 because that is the version that we used to validate our example workflows. We did not run into this issue during our validation, but we did not try to run single tasks (we only validated our example workflows). I plan to revisit the NetApp AI Control Plane solution soon in order to update the example workflows to incorporate the data science toolkit, as we discussed in a separate thread. At that time, I will validate our example workflows using the latest stable version of Airflow and will update the documentation accordingly.
